File not found

```diff
@@ -0,0 +1,7 @@
+@Article{,
+  author = {Test1-abc},
+  title  = {Test1},
+  month  = may,
+}
+
+@Comment{jabref-meta: databaseType:bibtex;}
```

Comment:
The path of the file is very strange. I think you accidently commited the file and it should not be part of the PR?